inttest/common/pod.go

```diff
@@ -1,3 +1,19 @@
+/*
+Copyright 2022 k0s authors
```

Comment:
Any idea why this has been missed by golangci-lint? Maybe the inttests aren't included?